code/game/gamemodes/clock_cult/clock_cult.dm

```diff
@@ -40,25 +40,7 @@ This file's folder contains:
 	return M && istype(M) && M.mind && ticker && ticker.mode && (M.mind in ticker.mode.servants_of_ratvar)
 
 /proc/is_eligible_servant(mob/living/M)
-	if(!istype(M))
-		return 0
-	if(!M.mind)
```

Comment:
you're checking `!is_servant_of_ratvar()` which WILL NOT CHECK THAT, and will in fact return 0 if they lack a mind.